### PR TITLE
Patch for turks guns

### PR DESCRIPTION
**Pull request by Arkadyzz**
Guns work perfectly without an error, also my first time doing PR so hopefully I dont screw this up

### CODE REVIEW

File: Patches/turk's guns/Patch_Guns.xml [hunk 1]
```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+	<success>Always</success>
+	<mods>
+		<li>turk's guns</li>
+	</mods>
+	
+	<match Class="PatchOperationSequence">
```
Comment (N7Huntsman) [inline Patches/turk's guns/Patch_Guns.xml:10]: CE now uses some fairly standard melee stats for various "classes" of ranged weapons (long-gun, pistol, heavy) just to make things easier and to allow some patch operations to be combined. Please check out how it's set up in Weapons_Guns.xml under Patches > Core and format this one similiarly.

File: Patches/turk's guns/Patch_Guns.xml [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+            <aimedBurstShotCount>2</aimedBurstShotCount>
+		  </FireModes>
+		  <weaponTags>
+		    <li>IndustrialGunAdvanced</li>
+		  </weaponTags>
+		  <researchPrerequisite>GasOperation</researchPrerequisite>
+		</li>
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+		  <defName>Gun_HAMR</defName>
```
Comment (N7Huntsman) [inline Patches/turk's guns/Patch_Guns.xml:148]: I'd definitely patch this gun as a SCAR H (7.62 NATO) with a larger magazine. The SCAR H is patched already in the files, so it should be a pretty easy swap.

File: Patches/turk's guns/Patch_Guns.xml [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+		  </statBases>
+		  <costList>
+		  <Steel>150</Steel>
+		  <ComponentIndustrial>12</ComponentIndustrial>
+		  </costList>
+		  <Properties>
+		    <recoilAmount>0.77</recoilAmount>
+		    <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+			<hasStandardCommand>true</hasStandardCommand>
+			<defaultProjectile>Bullet_556x45mmNATO_FMJ</defaultProjectile>
```
Comment (N7Huntsman) [inline Patches/turk's guns/Patch_Guns.xml:252]: I think the heavy minigun should be 7.62 NATO, modelled after the actual minigun. For this one, I think you can probably copy-paste the stats for the vanilla minigun if you like.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Generally looks good, just a few specifics to tweak here and there.

File: Patches/turk's guns/Patch_Guns.xml (hunk 1 above)
Comment (Arkadyzz) [reply Patches/turk's guns/Patch_Guns.xml:10]: it is done

File: Patches/turk's guns/Patch_Guns.xml (hunk 3 above)
Comment (Arkadyzz) [reply Patches/turk's guns/Patch_Guns.xml]: sounds good

File: Patches/turk's guns/Patch_Guns.xml (hunk 2 above)
Comment (Arkadyzz) [reply Patches/turk's guns/Patch_Guns.xml:144]: I would keep the gun as is to fill the role of SAW but let me know if you insist on this

**Review by N7Huntsman — APPROVED** [review]
Looks good. There are a few tweaks I'll make.

**Pull request merged**